src/odemis/gui/conf/data.py

```diff
@@ -648,6 +648,14 @@
             },
         },
     },
+    "mbsem": {
+        "e-beam": {
+            "scale": {
```

Comment:
Please add comment, why is that necessary? What happens normally, if you don't add these lines?